.github/workflows/validate_linux_rt.yml

```diff
@@ -0,0 +1,34 @@
+name: Validate Linux RT Feed
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  validate_linux_rt:
+    name: Validate Linux RT Feed
```

Comment:
Maybe `Validate Linux RT Feed Package Version` so it's clearer what you're actually checking.

Reply:
Same for the name of the workflow.

Reply:
Yeah, I like that better. Updated both the workflow and job name to the name you suggested.